TramsDataApi.Test/Integration/TrustsIntegrationTests.cs

```diff
@@ -81,9 +101,9 @@ public void Dispose()
            //_dbContext.Database.CurrentTransaction.Dispose();
```

Comment:
Potentially worth clearing this up and once this PR is wrapped look at organising the test setup so tests operate in a transaction and are rolled back (if you haven't already, this being now commented out tells me maybe that has been done)

Reply:
Yeah, we tried transaction-based tests and they didn't seem to play nicely with our setup and I couldn't quite figure out why

Reply:
Update: figured it out. Added transactions to `DbFixture`, ran tests a few times and checked in datagrip and looks like it's rolling them back properly 🎉